diff --git a/ITKImageProcessingFilters/ITKSaltAndPepperNoiseImage.cpp b/ITKImageProcessingFilters/ITKSaltAndPepperNoiseImage.cpp
index 8d88251..4a9ba48 100644
--- a/ITKImageProcessingFilters/ITKSaltAndPepperNoiseImage.cpp
+++ b/ITKImageProcessingFilters/ITKSaltAndPepperNoiseImage.cpp
@@ -19,7 +19,8 @@
 

 #include "ITKImageProcessing/ITKImageProcessingFilters/Dream3DTemplateAliasMacro.h"
-
+#undef DREAM3D_USE_RGBA
+#define DREAM3D_USE_RGBA 1
 // Include the MOC generated file for this class
 #include "moc_ITKSaltAndPepperNoiseImage.cpp"
 
diff --git a/ITKImageProcessingFilters/ITKSmoothingRecursiveGaussianImage.cpp b/ITKImageProcessingFilters/ITKSmoothingRecursiveGaussianImage.cpp
index 5ed7c22..384c64b 100644
--- a/ITKImageProcessingFilters/ITKSmoothingRecursiveGaussianImage.cpp
+++ b/ITKImageProcessingFilters/ITKSmoothingRecursiveGaussianImage.cpp
@@ -19,6 +19,8 @@
 

 #include "ITKImageProcessing/ITKImageProcessingFilters/Dream3DTemplateAliasMacro.h"
+#undef DREAM3D_USE_RGBA
+#define DREAM3D_USE_RGBA 1
 
 // Include the MOC generated file for this class
 #include "moc_ITKSmoothingRecursiveGaussianImage.cpp"
